### ISSUE TITLE
Drop cache

### ISSUE THREAD

**Issue by annevk** [w3c/webappsec-clear-site-data#68]
As discussed in https://github.com/privacycg/storage-partitioning/issues/11 it's complicated already and in a partitioned future even less desirable. There's also very little advocacy in favor of it that I've heard raising this topic with people. Folks primarily care about storage (and that clearing all of it, including service workers and such).

We'll likely do this in Firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=1671182.

**Comment by annevk** [comment]
Update: Firefox dropped support for this as discussed.

**Comment by o-t-w** [comment]
From a developer point of view the status of all of this is now a bit confusing.

- Firefox purposefully removed support for "cache"
- Chrome/Edge does support "cache" and clears all caches
- Safari Technology Preview does support "cache" but only clears bfcache?

Do I have that right?

**Comment by miketaylr** [comment]
> * Safari Technology Preview does support "cache" but only clears bfcache?

I haven't done any testing on TP, but I did see they just added support for CSD in these two commits:

https://github.com/WebKit/WebKit/commit/c65efd2b205ccaef913dff861a6e354dbdb9a212
https://github.com/WebKit/WebKit/commit/5e99c8f621c3966f7de0b5d4c407c26d458bbed1

Given https://github.com/WebKit/WebKit/commit/c65efd2b205ccaef913dff861a6e354dbdb9a212#diff-f978e2275336c983f36efd6d218fe5bd8c7d3f4aa9932618470bc6c73762ea99R396, I would expect it to not just be restricted to bfcache.

**Comment by cdumez** [comment]
> > * Safari Technology Preview does support "cache" but only clears bfcache?
> 
> I haven't done any testing on TP, but I did see they just added support for CSD in these two commits:
> 
> [WebKit/WebKit@c65efd2](https://github.com/WebKit/WebKit/commit/c65efd2b205ccaef913dff861a6e354dbdb9a212) [WebKit/WebKit@5e99c8f](https://github.com/WebKit/WebKit/commit/5e99c8f621c3966f7de0b5d4c407c26d458bbed1)
> 
> Given [WebKit/WebKit@c65efd2#diff-f978e2275336c983f36efd6d218fe5bd8c7d3f4aa9932618470bc6c73762ea99R396](https://github.com/WebKit/WebKit/commit/c65efd2b205ccaef913dff861a6e354dbdb9a212#diff-f978e2275336c983f36efd6d218fe5bd8c7d3f4aa9932618470bc6c73762ea99R396), I would expect it to not just be restricted to bfcache.

Correct. Unless there is a bug, my intention when implementing this was that `cache` would clear both the HTTP disk cache AND the bfcache in WebKit.

**Comment by miketaylr** [comment]
Mozilla is adding this back: https://groups.google.com/a/mozilla.org/g/dev-platform/c/tTkLvudz9MA/m/SKdx62IeBgAJ

So I think dropping it is even more of a non-starter. Maybe this should be renamed to "spec what to do with partitioned cache".
